vite.config.ts

```diff
@@ -9,6 +9,9 @@ export default defineConfig({
 			"@": path.resolve(__dirname, "src"),
 		},
 	},
+	optimizeDeps: {
+		include: ["@html-eslint/eslint-plugin"],
```

Comment:
html-eslint is written in commonjs. (https://vite.dev/config/dep-optimization-options)